dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -158,7 +160,7 @@ public void initialize(final URL location, final ResourceBundle resources) {
             ds3Common.setDeepStorageBrowserPresenter(deepStorageBrowserPresenter);
             try {
                 //open default session when DSB launched
-                savedSessionStore.openDefaultSession(ds3SessionStore);
+                savedSessionStore.openDefaultSession(ds3SessionStore, resourceBundle);
```

Comment:
Why is the resource bundle needed here?  Why is it  not injected into the savedSessionStore when it is created?